api/v1/_submissions/BackendSubmissionsHandler.inc.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file api/v1/_submissions/BackendSubmissionsHandler.inc.php
```

Comment:
Personally I believe we should either use /submissions for public and private submissions endpoints or split them by having backend endpoints under /backend (or another name). Having /submissions and /_submissions can be confusing and misleading.  @asmecher what do you think?